torch/csrc/jit/passes/quantization.cpp

```diff
@@ -828,5 +828,44 @@ graph(%self, %x):
   }
 }
 
+void FoldQuantize(
+    script::Module& module,
+    const std::string& method_name) {
+  const std::string pattern = R"(
+graph(%self, %scale, %zero_point, %dtype):
+   %weight = prim::GetAttr[name="weight"](%self)
+   %weight_quant = aten::quantize_linear(%weight, %scale, %zero_point, %dtype)
+   return (%weight_quant))";
+  Graph pattern_graph;
+  std::unordered_map<std::string, Value*> vmap;
+  script::parseIR(pattern, &pattern_graph, vmap);
+  auto method = module.get_method(method_name);
+  auto graph = method.graph();
+  auto matches = findPatternMatches(pattern_graph, *graph);
+  for (const auto& match: matches) {
+    auto match_vmap = match.values_map;
+    auto* weight = match_vmap.at(vmap.at("weight"));
+    TORCH_INTERNAL_ASSERT(
```

Comment:
You don't need to assert this - this is guaranteed by the API of subgraph matcher.